x-pack/plugins/alerting/server/lib/get_es_request_timeout.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger } from '@kbn/core/server';
+import { parseDuration } from '../../common';
+
+export function getEsRequestTimeout(logger: Logger, timeout?: string): number | undefined {
+  if (!timeout) {
+    return undefined;
```

Comment:
Does the client use the default timeout when we return `undefined`?

Reply:
yes, it will use the default timeout